src/app/core/services/common/auth.service.ts

```
@@ -34,6 +36,23 @@ export class AuthService {
     return this.apiService.post('/auth/login_with_refresh_token/', { refresh_token: refreshToken });
   }
 
+  storeTokens(appKey: IntegrationAppKey, tokens: Tokens) {
+    let integrationTokens: IntegrationTokensMap | undefined = this.storageService.get('integration-tokens');
+    if (integrationTokens) {
+      integrationTokens[appKey] = tokens;
+    } else {
+      integrationTokens = {[appKey]: tokens};
+    }
+
+    console.log('[x] storing', appKey, {integrationTokens});
```

ESLint

[error] Unexpected console statement.

(no-console)